### PR TITLE
Add reconfigure/reschedule actions

### PR DESCRIPTION
**Pull request by DmZ**

### CODE REVIEW

**Comment by dtkachenko** [comment]
Please test and merge that as soon as possible.

File: qubell/cli/__main__.py [hunk 1]
```diff
@@ -671,6 +671,49 @@ def _get_module(modules, name):
     _describe_instance(inst, True)
 
 
+@instance_cli.command("reconfigure", help="Reconfigure running instance")
+@click.option("--parameter", default=False, type=(unicode, unicode), multiple=True, help="Parameter value")
```
Comment (vasichkin) [inline qubell/cli/__main__.py]: @click.option("--version", default=None, help="Target manifest version")
@click.option("--revision", default=None, help="Target revision")

File: qubell/cli/__main__.py [hunk 2]
```diff
@@ -671,6 +671,49 @@ def _get_module(modules, name):
...
+@click.argument("instance")
+def reconfigure_instance(instance, parameter):
+    platform = _get_platform()
+    org = platform.get_organization(QUBELL["organization"])
+    inst = org.instances[instance]
```
Comment (vasichkin) [inline qubell/cli/__main__.py]: org.get_instance(instance) is little better

File: qubell/cli/__main__.py [hunk 3]
```diff
@@ -671,6 +671,49 @@ def _get_module(modules, name):
...
+            module = _get_module({'submodules': submodules}, module_name)
+            module['parameters'][module_param_name] = param_value
+        else:
+            parameters[param_name] = param_value
+
+    inst.reconfigure(parameters=parameters, submodules=submodules)
+    _describe_instance(inst, True)
+
+
+@instance_cli.command("reschedule", help="Reschedule destroy workflow for instance")
```
Comment (vasichkin) [inline qubell/cli/__main__.py]: Looks like should be more generic - reschedule workflow with 'destroy' as default workflow name

File: qubell/cli/__main__.py (hunk 2 above)
Comment (DmZ) [inline qubell/cli/__main__.py]: Maybe, but you need to change first this in all other instance actions (for unification).

**Comment by vasichkin** [comment]
Ok to merge after check

**Comment by DmZ** [comment]
@vasichkin verified reconfigure/reschedule on staging test organization

**Pull request merged**